benchmarks/cpp/nvfuser/timm.cpp

```diff
@@ -0,0 +1,431 @@
+
+  std::vector<bool> bcast_pattern0({true, true, false});
+  std::vector<bool> bcast_pattern1({false, false, true});
+
+  auto t4 = castOp(DataType::Float, t3);
+  auto t6 = set(t5);
+  auto t7 = broadcast(t6, bcast_pattern0);
+  auto t8 = add(t4, t7);
+  auto t9 = randlike(t8);
+  auto d34 =
+      sub(IrBuilder::create<Double>(1.0), IrBuilder::create<Double>(0.0));
+  auto t10 = lt(t9, d34);
+  auto t11 = castOp(DataType::Float, t10);
+  auto t12 = mul(t8, t11);
+  auto b36 = eq(d34, IrBuilder::create<Double>(0.0));
+  auto d37 = castOp(DataType::Double, b36);
+  auto d38 = add(d37, d34);
+  auto d40 = div(IrBuilder::create<Double>(1.0), d38);
+  auto t13 = mul(t12, d40);
+  auto t14 = set(t13);
+  auto t15 = add(t2, t14);
+  auto t16 = set(t15);
+  auto t36 = sum(t16, {2});
+  auto d151 = castOp(DataType::Double, t2->axis(2)->extent());
+  auto d152 = mul(IrBuilder::create<Double>(1.0), d151);
+  auto t19 = div(t36, d152);
+  auto t22 = broadcast(t19, bcast_pattern1);
+  auto t23 = sub(t16, t22);
+  auto t37 = mul(t23, t23);
+  auto t20 = sum(t37, {2});
+  auto t24 = broadcast(t20, bcast_pattern1);
+  auto d95 = castOp(DataType::Double, t2->axis(2)->extent());
+  auto d96 = mul(IrBuilder::create<Double>(1.0), d95);
+  auto d105 = reciprocal(d95);
+  auto t25 = mul(t24, d105);
+  auto t26 = add(t25, IrBuilder::create<Double>(1e-6));
+  auto t27 = rsqrt(t26);
+  auto t28 = mul(t23, t27);
+  auto t17 = set(t1);
+  auto t29 = broadcast(t17, bcast_pattern0);
+  auto t30 = mul(t28, t29);
+  auto t18 = set(t0);
+  auto t31 = broadcast(t18, bcast_pattern0);
+  auto t32 = add(t30, t31);
+  auto t33 = set(t32);
+  auto t34 = castOp(DataType::Half, t33);
+
+  fusion->addOutput(t16); // full 3d float
+  fusion->addOutput(t10); // full 3d bool
+  fusion->addOutput(t22); // bcast last dim float
+  fusion->addOutput(t27); // bcast last dim float
+  fusion->addOutput(t18); // passthrough t0 float
+  fusion->addOutput(t17); // passthrough t1 float
+  fusion->addOutput(t34); // full 3d half
+}
+
+static void NvFuserScheduler_TIMM_vit_base_patch16_224_kernel5(
+    benchmark::State& benchmark_state,
+    FusionExecutorCache* fusion_executor_cache,
+    void* null) {
+  std::vector<int64_t> input_shape{
+      benchmark_state.range(0),
+      benchmark_state.range(1),
+      benchmark_state.range(2)};
+
+  at::manual_seed(0);
+  auto fp16_options = at::TensorOptions().dtype(at::kHalf).device(at::kCUDA, 0);
+  auto fp32_options =
+      at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+
+  auto t2 = at::randn(input_shape, fp32_options);
+  auto t5 = at::randn({input_shape[2]}, fp32_options);
+  auto t3 = at::randn(input_shape, fp16_options);
+  auto t0 = at::randn({input_shape[2]}, fp32_options);
+  auto t1 = at::randn({input_shape[2]}, fp32_options);
+
+  std::vector<c10::IValue> aten_inputs({t2, t5, t3, t0, t1});
+  runBenchmarkIterations(benchmark_state, fusion_executor_cache, aten_inputs);
+
+  // inputs: full float, full half, inner dim 3xfloat
```

Comment:
These comments would be useful for the bandwidth calculations in the other tests.